test/common/http/conn_manager_utility_test.cc

```diff
@@ -768,6 +768,9 @@ TEST_F(ConnectionManagerUtilityTest, MtlsSetForwardClientCert) {
   EXPECT_CALL(ssl, uriSanPeerCertificate()).WillRepeatedly(Return(peer_uri_sans));
   std::string expected_pem("%3D%3Dabc%0Ade%3D");
   EXPECT_CALL(ssl, urlEncodedPemEncodedPeerCertificate()).WillOnce(ReturnRef(expected_pem));
+  std::string expected_chain_pem("%3D%3Dlmn%0Aop%3D");
```

Comment:
Nitpicking, but `urlEncodedPemEncodedPeerCertificateChain()` returns entire certificate chain (including leaf certificate), so `expected_chain_pem` should start with `expected_pem`.